client/elasticsearch/v710/client.go

```diff
@@ -186,6 +185,30 @@ func (cli *ESClient) AddDocument(ctx context.Context, indexName, documentID stri
 	return nil
 }
 
+// Msearch allows to execute several search operations in one request.
+// See full documentation at https://www.elastic.co/guide/en/elasticsearch/reference/master/search-multi-search.html.
+func (cli *ESClient) MultiSearch(ctx context.Context, indices []string, document []byte) error {
+	req := esapi.MsearchRequest{
+		Index: indices,
```

Comment:
Note that the index is supplied as part of the header line for each one of the multiple searches. There is no need to set here.

Reply:
Or indeed to take it in as a function argument.

Reply:
Fixed in update 1.